portfolio/src/main/webapp/index.html

```diff
@@ -22,7 +22,7 @@ <h1>KIZITO AKWOTIGA</h1>
                     <p>COMPUTER ENGINEER</p>
                     <a href="#" class="side-menu">ABOUT ME</a>
                     <a href="#" class="side-menu">RESUME</a>
-                    <a href="#" class="side-menu">GET IN TOUCH</a>
+                    <a href="./contactMe.html" class="side-menu">GET IN TOUCH</a>
```

Comment:
Is this correctly working ?
I believe the href value should just be "/contactMe.html"

Reply:
Surprisingly it works but will check your suggestion as well.